gyft.py

```diff
@@ -5,21 +5,12 @@
 import re
 import json
 import getpass
-
-#### Parsing from commmand line
-import argparse
-parser = argparse.ArgumentParser()
-parser.add_argument("-u", "--user", help="ERP Username/Login ID")
-args = parser.parse_args()
-if args.user is None:
-	args.user = input("Enter you Roll Number: ")
-erp_password = getpass.getpass("Enter your ERP password: ")
-
-#### Parsing ends
+import iitkgp_erp_login.erp as erp
 
 ERP_HOMEPAGE_URL = 'https://erp.iitkgp.ac.in/IIT_ERP3/'
 ERP_LOGIN_URL = 'https://erp.iitkgp.ac.in/SSOAdministration/auth.htm'
 ERP_SECRET_QUESTION_URL = 'https://erp.iitkgp.ac.in/SSOAdministration/getSecurityQues.htm'
+ERP_OTP_URL = 'https://erp.iitkgp.ac.in/SSOAdministration/getEmilOTP.htm' # blame ERP for the typo
```

Comment:
```suggestion
```
The package already includes these, just check once whether they are used else where or not ( I don't see them )

Reply:
You're right, they're not required anymore, I hadn't noticed it.